bin/objectbox_model_generator/lib/src/generator.dart

```diff
@@ -78,7 +78,8 @@ class EntityGenerator extends GeneratorForAnnotation<obx.Entity> {
             flags |= OBXPropertyFlag.ID;
             hasIdProperty = true;
           } else if (annotType == "Property") {
-            // nothing special
+            fieldType = annotVal.getField("type")?.toIntValue();
+            flags |= annotVal.getField("flag")?.toIntValue();
```

Comment:
crashes if `flag` annotation is not present.
maybe?
```suggestion
            flags |= annotVal.getField("flag")?.toIntValue()?? 0;
```

Reply:
I fixed all of that, please check again.